pkg/system/phase4_configuring.go

```diff
@@ -1523,3 +1529,136 @@ func (r *Reconciler) ReconcileNamespaceStores(namespaceResources []nb.NamespaceR
 	}
 	return nil
 }
+
+// ReconcileNoobaaDeletionPolicy reconciles the deletion policy of a Noobaa resource
+// and configures the noobaa admission controller appropriately.
+func (r *Reconciler) ReconcileNoobaaDeletionPolicy() error {
+	noobaaAdmissionEnabled := os.Getenv("ENABLE_NOOBAA_ADMISSION")
+	if noobaaAdmissionEnabled == "" || strings.ToLower(noobaaAdmissionEnabled) == "false" {
```

Comment:
why not use strings.ToLower(noobaaAdmissionEnabled) != "true"?

Reply:
Oh, actually that was just because I keep the branches slim 😅 . Have made the change though ✔️ .

Reply:
but ```if strings.ToLower(noobaaAdmissionEnabled) != "true"```  evaluates to true you should return